sig-cloud-provider/README.md

```diff
@@ -34,9 +34,41 @@ The Chairs of the SIG run operations and processes governing the SIG.
 ## Subprojects
 
 The following subprojects are owned by sig-cloud-provider:
```

Comment:
About listing the sub-projects, instead of listing alphabetically, is it a good idea to group sub-projects of a particular cloud provider under a sub-section by the slack names that we decided e.g. list all aws sub-projects under something like cloud-provider-aws sub-section, for a better readability as the list is bigger now and probably more sub-projects will be added in future? Just a thought. Thanks!

Reply:
Yep, in the proposal, we mention that each provider would migrate in as `provider-<provider>`, with the associate [sub-]subprojects within e.g.,

**provider-azure**
- https://raw.githubusercontent.com/kubernetes/cloud-provider-azure/master/OWNERS
- https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/master/OWNERS

Reply:
OK, awesome! Thanks @justaugustus !!